simulation/src/LeiosProtocol/Short.hs

```diff
@@ -111,6 +111,10 @@ data LeiosConfig = forall p. IsPipeline p => LeiosConfig
   , variant :: LeiosVariant
   , headerDiffusionTime :: NominalDiffTime
   -- ^ Δ_{hdr}.
+  , leiosLateIbInclusion :: Bool
+  -- ^ Whether an EB also includes IBs from the two previous iterations.
+  --
+  -- TODO Merely one previous iteration if 'pipeline' is 'SingleVote'?
```

Comment:
The Haskell simulator is configurable to use a single stage for voting )Vote) instead of two stages (VoteSend and VoteRecv).

The Short-to-Full spec only considers the two stage variant, when it specifies to include 3 iterations' IBs in each EB.

Would the Short-to-Full spec only include 2 iterations' IBs if Vote were a single stage? Should the Haskell simulator do that? @pagio @SupernaviX